aspnet/publishing/iis.rst

```diff
@@ -1,116 +1,187 @@
 
-.. image:: pubIIS/_static/config-edit.png
-.. image:: pubIIS/_static/unlock.png
+Configure the Website in IIS
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- Set the application pool to **No Managed Code**. ASP.NET Core runs in a separate process and manages the runtime.
+#. In **IIS Manager**, create a new website. Provide a **Site name** and set the **Physical path** to the application's assets folder that you created. Provide the **Binding** configuration and create the website. [See the screenshots below.]
+#. Set the application pool to **No Managed Code**. [See the screenshot below.] ASP.NET Core runs in a separate process and manages the runtime.
 
- .. image:: pubIIS/_static/appPool.PNG
+.. note:: If you change the default identity of the application pool from **ApplicationPoolIdentity** to some other user account, make sure you configure the correct permissions for the user identity to use the application's assets folder.
 
+Open the **Add Website** window.
 
-Supported operating systems
------------------------------------
+	.. image:: pubIIS/_static/addwebsitecontextmenu.png
+	
+Configure the website.
 
-The following operations systems are supported:
+	.. image:: pubIIS/_static/addwebsite.png
+	
+In the **Application Pools** panel, open the **Edit Application Pool** window by right-clicking on the website's application pool and selecting **Basic Settings...** from the popup menu.
 
-- Windows 7 and newer
-- Windows 2008 R2 and newer
+	.. image:: pubIIS/_static/basicsettingscontextmenu.png
+	
+Set the **.NET CLR version** to **No Managed Code**.
+
+	.. image:: pubIIS/_static/editapppool.png
+
+Create a Data Protection Registry Hive
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Data Protection keys used by ASP.NET applications are stored in registry hives external to the applications. To persist the keys for a given application, you must create a registry hive for the application's application pool.
+
+For standalone IIS installations, you may use the `Data Protection Provision-AutoGenKeys.ps1 PowerShell script <https://github.com/aspnet/DataProtection/blob/dev/Provision-AutoGenKeys.ps1>`_ for each application pool used with an ASP.NET Core application. The keys will be persisted in the registry.
+
+In web farm scenarios, an application can be configured to use a UNC path to store its data protection key ring. By default, the data protection keys are not encrypted. You can deploy an x509 certificate to each machine to encrypt the key ring. See :ref:`Configuring Data Protection <data-protection-configuring>` for details.
+
+.. include:: ./dataProtectionWarning.txt
+
+Common Errors
```

Comment:
Will prune the purely IIS ones ... **Arrrrrrrrrrrrrr** :-1: but your call, of course.